hrm-service/src/swagger.js

```diff
@@ -2,14 +2,16 @@ const swaggerUi = require('swagger-ui-express');
 const yaml = require('js-yaml');
 const fs = require('fs');
 
-const swaggerDocument = yaml.safeLoad(fs.readFileSync(`${__dirname}/../swagger.yaml`, 'utf8'));
-
-const runWhenNotProduction = app => {
+const runWhenNotProductionOrTest = app => {
   const isProduction = process.env.NODE_ENV && process.env.NODE_ENV === 'production';
+  const isTest = process.env.NODE_ENV && process.env.NODE_ENV === 'test';
 
-  if (!isProduction) {
+  if (!isProduction && !isTest) {
```

Comment:
Again, a separate swagger flag that falls back to behaviour based on environment would be preferable here